websites/P/Pokéflix/iframe.ts

```diff
@@ -0,0 +1,14 @@
+const iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {
+	if (document.querySelector("video")) {
+		const video = document.querySelector<HTMLVideoElement>("video");
+		if (!isNaN(video?.duration)) {
```

Comment:
```suggestion
		if (!isNaN(video.duration)) {
```

or this:
```
const iframe = new iFrame(),
	video = document.querySelector<HTMLVideoElement>("video");

iframe.on("UpdateData", async () => {
	if (video) {
		if (!isNaN(video.duration)) {
			iframe.send({
				duration: video.duration,
				currentTime: video.currentTime,
				paused: video.paused,
			});
		}
	}
});
```

Reply:
> or this:
> 
> ```
> const iframe = new iFrame(),
> 	video = document.querySelector<HTMLVideoElement>("video");
> 
> iframe.on("UpdateData", async () => {
> 	if (video) {
> 		if (!isNaN(video.duration)) {
> 			iframe.send({
> 				duration: video.duration,
> 				currentTime: video.currentTime,
> 				paused: video.paused,
> 			});
> 		}
> 	}
> });
> ```

```
pmd Pokéflix/iframe.ts:2:2 - Error TS2451: Cannot redeclare block-scoped variable 'video'.
pmd Pokéflix/presence.ts:28:5 - Error TS2451: Cannot redeclare block-scoped variable 'video'.
pmd Pokéflix/presence.ts:46:3 - Error TS2588: Cannot assign to 'video' because it is a constant.
pmd Failed to compile with 3 errors!
```